README.md

```diff
@@ -197,43 +207,17 @@ ChartMogul.SubscriptionEvent.deleteWithParams(config, {
     id: 1
   }
 })
+ChartMogul.SubscriptionEvent.all(config, query)
 ```
 
 ### Enrichment API
 
 Available methods in Enrichment API:
 
-
 #### [Customers](https://dev.chartmogul.com/docs/retrieve-customer)
```

Comment:
Shouldn't the title and link also be changed to [Search for Customers](https://dev.chartmogul.com/reference/search-for-customers) here?

Reply:
This heading 4 is the higher level endpoint "family", or at least that's how the README look like now for the other endpoints, so it points to https://dev.chartmogul.com/reference/customers-1 instead. The URL was wrong.

Reply:
Hmm, but the code sample is about search, that's what confused me.